tests/performance-test/deploy/performance-test-tb.sh

```diff
@@ -8,12 +8,23 @@ CONCURRENT=${CONCURRENT:-1}
 
 oc delete job -l app=saf-performance-test || true
 
-for i in $(seq 1 ${CONCURRENT}); do
+oc delete configmap/saf-performance-test-collectd-config \
+          configmap/saf-performance-test-events-entry \
+          job/saf-perftest-notify || true
+
+oc create configmap saf-performance-test-collectd-config --from-file \
+        ./config/minimal-collectd.conf
 
-  oc create -f <(sed  -e "s/<<PREFIX>>/saf-perftest-${i}-/g;
-                          s/<<COUNT>>/${COUNT}/g;
-                          s/<<HOSTS>>/${HOSTS}/g;
-                          s/<<PLUGINS>>/${PLUGINS}/g;
-                          s/<<INTERVAL>>/${INTERVAL}/g"\
-                          performance-test-job-tb.yml.template)
+oc create configmap saf-performance-test-events-entry --from-file \
+          ./entrypoint.sh
+          
+oc create -f ./performance-test-job-events.yml.template
```

Comment:
This is a bit weird, historically speaking, having `performance-test-tb.sh` calling collectd-tg. Since you deleted the tg version of this script, then.... okay sure :)

Reply:
Agreed it is a little strange. Calling it 'notify' instead of collectd may keep that mental clash from happening